### ISSUE TITLE
It is incompatible with python-etcd3

### ISSUE THREAD

**Issue by zhuwenxing** [XuanYang-cn/pyetcd#10]
```
import etcd3
etcd = etcd3.client(host="10.101.28.233", port=2379)
leader = etcd.status().leader
print(leader)
>>> 
Member 17573030977508649259: peer urls: ['http://yanliang-cd-mmszh-etcd-1.yanliang-cd-mmszh-etcd-headless.chaos-testing.svc.cluster.local:2380'], client urls: ['http://yanliang-cd-mmszh-etcd-1.yanliang-cd-mmszh-etcd-headless.chaos-testing.svc.cluster.local:2379']
```

```
import pyetcd  as etcd3
etcd = etcd3.client(host="10.101.28.233", port=2379)
leader = etcd.status().leader
print(leader)
>>> 
Traceback (most recent call last):
  File "/Users/zilliz/workspace/etcd_learn/demo.py", line 6, in <module>
    leader = etcd.status().leader
             ^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pyetcd/client.py", line 342, in handler
    return payload(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pyetcd/client.py", line 708, in status
    for m in self.members:
  File "/usr/local/lib/python3.11/site-packages/pyetcd/client.py", line 1181, in members
    yield etcd3.members.Member(member.ID,
          ^^^^^
NameError: name 'etcd3' is not defined
```

**Comment by zhuwenxing** [comment]
in https://github.com/XuanYang-cn/pyetcd/blob/9f8ba8e31d3349ab54dbbebd7b851903821d277a/pyetcd/client.py#L1180

It should already be fixed in the main branch.
But that should be a bug in the source code of `etcd-sdk-python==0.0.1`.
Any plan to publish a package?

**Comment by XuanYang-cn** [comment]
@zhuwenxing should be fixed in etcd-sdk-python==0.0.2
